workers.py

```diff
@@ -117,3 +117,75 @@ def worker() -> None:
 
     thread = Thread(target=worker)
     thread.start()
+
+
+def topology_client_download(topology_client, entity: str) -> list:
+    """Download a list for a Topological Inventory Entity."""
+    return topology_client.query_path(entity)
```

Comment:
Why do we need this function? Can't the `query_path` be called directly? What's the benefit?